skimage/measure/_regionprops.py

```diff
@@ -123,7 +123,7 @@ def bbox(self):
                      [self._slice[i].stop for i in range(self._ndim)])
 
     def bbox_area(self):
-        return self._label_image.size
+        return self.convex_image.size
```

Comment:
>Erm, convex_image is a computationally expensive feature to compute!

Exactly my thinking! I haven't checked in details, but I remeber that `regionprops` has a lot of lazy computations inside.